<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Find Rack</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>1920</width>
     <height>1080</height>
    </size>
   </property>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1921</width>
      <height>1051</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>home.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="arm_lbl">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>810</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">.QLabel {
    border: 2px solid transparent; /* Initially no visible border */
}

.QLabel:hover {
    border: 8px solid #53D287; /* Blue border on hover */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="takeoff_lbl">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>810</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">.QLabel {
    border: 2px solid transparent; /* Initially no visible border */
}

.QLabel:hover {
    border: 8px solid #53D287; /* Blue border on hover */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="land_lbl">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>810</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">.QLabel {
    border: 2px solid transparent; /* Initially no visible border */
}

.QLabel:hover {
    border: 8px solid #53D287; /* Blue border on hover */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="disarm_lbl">
    <property name="geometry">
     <rect>
      <x>1630</x>
      <y>810</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">.QLabel {
    border: 2px solid transparent; /* Initially no visible border */
}

.QLabel:hover {
    border: 8px solid #53D287; /* Blue border on hover */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="test_AI_code_lbl">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>940</y>
      <width>401</width>
      <height>91</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">.QLabel {
    border: 2px solid transparent; /* Initially no visible border */
}

.QLabel:hover {
    border: 8px solid #53D287; /* Blue border on hover */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="arm_lbl_7">
    <property name="geometry">
     <rect>
      <x>1450</x>
      <y>500</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">.QLabel {
    border: 2px solid transparent; /* Initially no visible border */
    transition: border 0.3s ease-in-out;
}

.QLabel:hover {
    border: 8px solid #53D287; /* Blue border on hover */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="up_lbl">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>370</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">.QLabel {
    border: 2px solid transparent; /* Initially no visible border */
}

.QLabel:hover {
    border: 8px solid #53D287; /* Blue border on hover */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="left_lbl">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>510</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">.QLabel {
    border: 2px solid transparent; /* Initially no visible border */
}

.QLabel:hover {
    border: 8px solid #53D287; /* Blue border on hover */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="right_lbl">
    <property name="geometry">
     <rect>
      <x>1450</x>
      <y>500</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">.QLabel {
    border: 2px solid transparent; /* Initially no visible border */
}

.QLabel:hover {
    border: 8px solid #53D287; /* Blue border on hover */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="down_lbl">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>620</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">.QLabel {
    border: 2px solid transparent; /* Initially no visible border */
}

.QLabel:hover {
    border: 8px solid #53D287; /* Blue border on hover */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
